src/functions/postTechRecords.ts

```diff
@@ -33,7 +32,7 @@ const postTechRecords = (event: any) => {
     vin,
     partialVin: populatePartialVin(vin),
     techRecord: techRec,
-    systemNumber: sysNum,
+    systemNumber: "",
```

Comment:
is this intended?

Reply:
yes, system number is now generated in the backend and it will have a proper value. If I remove it, typescript will complain because systemNumber is mandatory on ITechRecWrapper interface.